### PR TITLE
Improved openvswitch docs

### PR DESCRIPTION
**Pull request by nadvornik**

### CODE REVIEW

File: docs/Networking.asciidoc [hunk 1]
```diff
@@ -163,15 +200,45 @@ systemctl start os-autoinst-openvswitch.service
...
+shows the rules installed by os-autoinst-openvswitch in table=0
+
+* packets from tapX to br0 create additional rules in table=1
+* packets from br0 to tapX increase packet counts in table=1
+* empty output indicates a problem with os-autoinst-openvswitch service
+* zero packet count or missing rules in table=1 indicate problem with tap devices
+
+ipables -L -v
+
+As long as the SUT has access to externa network, there should be
```
Comment (aaannz) [inline docs/Networking.asciidoc]: typo: `externa` > `external`

**Comment by nadvornik** [comment]
typo fixed

**Comment by aaannz** [comment]
LGTM, @sysrich anything missing from your experience?

**Comment by sysrich** [comment]
Nope, and it all seems to be working after a reboot, merging with happiness!

**Pull request merged**